applications/ray/README.md

```diff
@@ -3,22 +3,58 @@
 This repository contains a Terraform template for running [Ray](https://www.ray.io/) on Google Kubernetes Engine.
 See the [Ray on GKE](/ray-on-gke/) directory to see additional guides and references.
 
+## Prerequisites
+
+1. GCP Project with following APIs enabled
```

Comment:
Can we enable these by default?

Reply:
I copied these pre-reqs from the jupyter readme. We do enable services in the terraform [here](https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/ray/main.tf#L39) so maybe this shouldn't be part of either README. Are you ok leaving it in for now?

Reply:
ok